packages/vite/src/node/plugins/worker.ts

```diff
@@ -264,6 +292,13 @@ export function webWorkerPlugin(config: ResolvedConfig): Plugin {
         url = injectQuery(url, WorkerFileId)
       }
 
+      if (isUrl) {
```

Comment:
Should we inline this line as:

```suggestion
      if (query.url != null) {
```

and remove line 263? I notice the `isUrl` variable isn't used anywhere else except this line.